elyra/pipeline/processor_kfp.py

```diff
@@ -169,6 +81,145 @@ def cc_pipeline():
 
         return None
 
+    def export(self, pipeline):
+        if pipeline.file_type not in ["tgz", "tar.gz", "zip", "yaml", "yml", "py"]:
+            self.log.warn("Pipeline file type not recognized...defaulting to " + pipeline.DEFAULT_FILETYPE)
+            pipeline_file_type = pipeline.DEFAULT_FILETYPE
```

Comment:
Should `DEFAULT_FILETYPE` be prefixed at all?  I figured it would be imported with `from parser import DEFAULT_FILETYPE`, then just used as itself.  This implies its an attribute of the `pipeline` instance, which I don't think it is.